src/rgw/rgw_ssd_driver.cc

```diff
@@ -328,8 +329,13 @@ int SSDDriver::restore_blocks_objects(const DoutPrefixProvider* dpp, ObjectDataC
                                                 ldpp_dout(dpp, 20) << "SSDCache: " << __func__ << "(): deleteMarker: " << deleteMarker << dendl;
                                             }
 
+                                            if (attrs.find(RGW_CACHE_ATTR_INVALID) != attrs.end()) {
+                                                invalidStr = attrs[RGW_CACHE_ATTR_INVALID].to_str();
+                                                ldpp_dout(dpp, 20) << "SSDCache: " << __func__ << "(): invalidStr: " << invalidStr << dendl;
+                                            }
+
                                             ldpp_dout(dpp, 20) << "SSDCache: " << __func__ << "(): calling func for: " << key << dendl;
-                                            obj_func(dpp, key, version, deleteMarker, size, creationTime, user, etag, bucket_name, bucket_id, obj_key, null_yield);
```

Comment:
Again the rebase has removed code that I added - I have changed `dirty` to `deleteMarker`.